network/engine.go

```diff
@@ -17,17 +17,17 @@ type Engine interface {
 	// Submit submits the given event from the node with the given origin ID
 	// for processing in a non-blocking manner. It returns instantly and logs
 	// a potential processing error internally when done.
-	Submit(originID flow.Identifier, event interface{})
+	Submit(channel Channel, originID flow.Identifier, event interface{})
```

Comment:
Shouldn't `SubmitLocal` and `ProcessLocal` also have channel? The `RPCEngine` in the access node has that definition.

Reply:
I changed it to remove Channel, due to what @AlexHentschel mentioned about how `SubmitLocal` and `ProcessLocal` are used between engines on the same node, and for those messages there isn't necessarily the notion of a channel

Reply:
I removed channel from `SubmitLocal` in the access node :)